src/diffpy/srfit/fitbase/fithook.py

```diff
@@ -150,7 +150,7 @@ def postcall(self, recipe, chiv):
             print("Variables")
             vnames = recipe.getNames()
             vals = recipe.getValues()
-            byname = lambda nv: sortKeyForNumericString(nv[0])
+            byname = lambda nv: sortKeyForNumericString(nv[0])  # noqa: E731
```

Comment:
Here I would put the lamda directly in the sorry and not assign it. Then we don't need the no qa

Reply:
Done